agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/ConfigurationBuilder.java

```diff
@@ -244,7 +239,10 @@ private static void logConfigurationWarnings(Configuration config) {
         config.sampling.requestsPerSecond = config.sampling.limitPerSecond;
       }
     }
-
+    if (config.authentication.clientSecret != null) {
+      configurationLogger.warn(
+          "We no longer support \"CLIENTSECRET\" typed of AAD authentication (from 3.5 GA). Please use \"user-assigned or system-assigned identify\" instead.");
```

Comment:
```suggestion
          "We no longer support \"CLIENTSECRET\" typed of AAD authentication (from 3.5.0 GA). Please use \"user-assigned or system-assigned identify\" instead.");
```

Reply:
If we ignore clientSecret configured by the customer and start up, we will not be able to send data to breeze anyways.  To avoid CRI we should throw and abort, IMO.